gulpfile.js

```diff
@@ -35,11 +35,11 @@ gulp.task("6to5", function () {
 });
 
 gulp.task("sass", function () {
-  return gulp.src("server/src/**/*.scss").pipe(sass()).pipe(gulp.dest("server/dist/css"));
+  return gulp.src("server/src/**/*.scss").pipe(sass()).pipe(gulp.dest("server/dist/static/css"));
```

Comment:
I don't like this extra level of folder nesting. I'd rather see all the static files move up to the top level /shared/ directory. But, we have another issue for that, so I guess we'll change the structure again later.